lib/errors.js

```diff
@@ -7,6 +7,14 @@ class NotYetImplementedError extends ES6Error {
   }
 }
 
+class NotImplementedError extends ES6Error {
+  constructor () {
+    super("That method is not implemented");
+    this.jsonwpCode = 15; // TODO use status package
```

Comment:
actually this is what we are supposed to do with not implemented:

> Every server implementing the WebDriver wire protocol must respond to every defined command. If an individual command has not been implemented on the server, the server should respond with a 501 Not Implemented error message. Note this is the only error in the Invalid Request category that does not return a 4xx status code.

so it shouldn't have a jsonwpCode at all